OHHTTPStubs/UnitTests/Test Suites/SwiftHelpersTests.swift

```diff
@@ -439,6 +439,18 @@ class SwiftHelpersTests : XCTestCase {
 
     XCTAssertFalse(matchesHeader)
   }
+    
+  func test_ohhttpStubs_httpBody() {
+#if swift(>=3.0)
+    var req = URLRequest(url: URL(string: "foo://bar")!)
+    req.httpBody = Data()
```

Comment:
Maybe we could use some non-empty data here to have a more robust test? 
Like `"Hello World".data(using: .utf8)` or something?